provisioners/src/test/java/org/jboss/intersmash/provision/helm/wildfly/EapHelmChartReleaseAdapterTest.java

```diff
@@ -202,4 +204,99 @@ public void verifyWildFlyDynamicallyFilledAdapterTest() {
 		Assertions.assertEquals(1, eap8HelmChartRelease.getDeploymentEnvironmentVariables().size());
 		Assertions.assertEquals(2, eap8HelmChartRelease.getInjectedImages().size());
 	}
+
+	/**
+	 * Verify that the EAP XP 5 adapter for the EAP XP 5values file POJO is properly storing and exposing the adaptee values
+	 */
+	@Test
+	public void verifyEapXp5DynamicallyFilledAdapterTest() {
```

Comment:
Maybe this method is a good chance for testing `BuildMode.BOOTABLE_JAR` (and the related image) ser/de, rather than `s2I` which we test already in existing methods. WDYT?

Reply:
Thanks, changed this and also added test solely for S2I to be sure.